edx_exams/apps/router/views.py

```diff
@@ -40,11 +41,21 @@ def patch(self, request, course_id):
                 exam['is_proctored'] = exam_type.is_proctored
                 exam['is_practice_exam'] = exam_type.is_practice
 
-        response_data, status = register_exams(course_id, exam_list)
+        response_data, response_status = register_exams(course_id, exam_list)
 
         return JsonResponse(
             data=response_data,
-            status=status,
+            status=response_status,
+            safe=False,
+        )
+
+    def get(self, request, course_id):  # pylint: disable=unused-argument
```

Comment:
Needed to add corresponding view in the middleware. Left this as an essentially empty response for now, but can also add support for the legacy view if needed.

Reply:
interesting, good call that we needed this.